tests/chainer_tests/training_tests/triggers_tests/test_interval_trigger.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import division
+
+import unittest
+
+from chainer import testing
+from chainer import training
+
+
+class DummyUpdater(training.Updater):
+
+    def __init__(self, iters_per_epoch):
+        self.iteration = 0
+        self.iters_per_epoch = iters_per_epoch
+
+    def finalize(self):
+        pass
+
+    def get_all_optimizers(self):
+        return {}
+
+    def update(self):
+        self.iteration += 1
+
+    @property
+    def epoch(self):
+        return self.iteration // self.iters_per_epoch
+
+    @property
+    def epoch_detail(self):
+        return self.iteration / self.iters_per_epoch
+
+    @property
+    def is_new_epoch(self):
+        return 0 <= self.iteration % self.iters_per_epoch < 1
```

Comment:
No, because `self.iters_per_epoch` may not be an integer (in this dummy iterator, which replicates the behavior of the real PTB iterator where iterations don’t evenly divide epochs)